src/main/java/mate/academy/repository/MySqlConnectionManager.java

```diff
@@ -0,0 +1,31 @@
+package mate.academy.repository;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+import java.util.Properties;
+
+public class MySqlConnectionManager {
+    private static final String DB_URL = "jdbc:mysql://localhost:3306/library";
+    private static final Properties DB_PROPERTIES;
+
+    static {
+        DB_PROPERTIES = new Properties();
+        DB_PROPERTIES.put("user", "test");
+        DB_PROPERTIES.put("password", "StrongSecret1234");
+
+        try {
+            Class.forName("com.mysql.cj.jdbc.Driver");
+        } catch (ClassNotFoundException e) {
+            throw new RuntimeException("MySQL JDBC driver not found", e);
+        }
+    }
+
+    public static Connection getConnection() {
+        try {
+            return DriverManager.getConnection(DB_URL, DB_PROPERTIES);
+        } catch (SQLException e) {
+            throw new RuntimeException("Cannot get connection", e);
+        }
+    }
```

Comment:
Maybe you shouldn't catch it there, for me looks better to catch this in Dao in try-with-resources